webpack.prod.js

```diff
@@ -21,6 +21,7 @@ module.exports = merge(commons, {
           path.resolve(__dirname, "src"),
           path.resolve(__dirname, "node_modules/esn-frontend-common-libs"),
           path.resolve(__dirname, "node_modules/esn-frontend-calendar"),
+          path.resolve(__dirname, "node_modules/esn-frontend-inbox"),
```

Comment:
Don't you need this also?

```suggestion
          path.resolve(__dirname, "node_modules/esn-frontend-inbox"),
          path.resolve(__dirname, "node_modules/esn-frontend-inbox-james"),
```

Reply:
actually not, I can `npm run serve:prod` without any problem.
But I guess we can just add it to be safe

Reply:
We should only add it when it breaks the prod mode